### ISSUE TITLE
Ignore-checks is not properly working in config file

### ISSUE THREAD

**Issue by antoniorodriguezmoreno** [aws-cloudformation/cfn-lint#2194]
*cfn-lint version: (`cfn-lint --version`)*
0.56.3

*Description of issue.*
I want to check several directories and there content if there is a malformed CF template and I don´t know if those files even exist, so I want to ignore the E0000 error, so that my pipeline will succeed.
If I use the following command:      
```bash
cfn-lint ./*.yml --ignore-templates ./.gitlab-ci.yml -r eu-central-1 -i E0000 -I ; cfn-lint ./*.yaml -r eu-central-1 -i E0000 -I ; cfn-lint ./*.json -r eu-central-1 -i E0000 -I .
```
I will receive following logs: 
```
2021-12-28 14:54:18,622 - cfnlint.runner - INFO - Run scan of template ./Amazon-S3-Bucket.yml
2021-12-28 14:54:20,870 - cfnlint.decode - ERROR - Template file not found: ./*.yaml
2021-12-28 14:54:20,870 - cfnlint.decode - ERROR - Template file not found: ./*.yaml
2021-12-28 14:54:23,071 - cfnlint.decode - ERROR - Template file not found: ./*.json
2021-12-28 14:54:23,071 - cfnlint.decode - ERROR - Template file not found: ./*.json
```
 And my pipeline succeeds as it should. 
 
 But if I do basically the same with the following command and the corresponding config-file, I still get the E0000 Error and my pipeline fails...
```bash
cfn-lint --config-file '.linter-config/.cfnlintrc.yml'
```

```yaml
templates:
  - ./**/*.yml
  - ./**/*.yaml
  - ./**/*.json
ignore_templates:
  - ./.gitlab-ci.yml
  - ./.linter-config/*
  - ./full-linter.yml
  - ./common/*
include_checks:
  - I
  - r eu-central-1
  - i E0000
```

Which has following output and will result in an exit code in a git-lab pipeline:
```
2021-12-28 14:50:47,057 - cfnlint.decode - ERROR - Template file not found: ./**/*.json
2021-12-28 14:50:47,057 - cfnlint.decode - ERROR - Template file not found: ./**/*.yaml
E0000 Template file not found: ./**/*.json
./**/*.json:1:1
E0000 Template file not found: ./**/*.yaml
./**/*.yaml:1:1
```

Why is the error even put out, even though I excluded it in the config file?

**Comment by PatMyron** [comment]
```yaml
include_checks:
  - I
  - r eu-central-1
  - i E0000
```
should be:
```yaml
include_checks:
  - I
regions:
  - eu-central-1
ignore_checks:
  - E0000
```

**Comment by antoniorodriguezmoreno** [comment]
@PatMyron thanks for the hint! 
Now it works as intended. :)
